frontend/src/components/ForgotPassword/ForgotPassword.tsx

```diff
@@ -9,33 +21,113 @@ import {
+      return;
+    }
+    setNotification(null);
+  };
+
+  const sendEmail = async (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    setLoading(true);
+
+    try {
+      const { data } = await axios.post(
+        `${REACT_APP_API_URI}/forgot-password`,
+        {
+          email
+        }
+      );
+      setEmail('');
+      setNotification({ type: 'success', message: data.message });
+    } catch (error: any) {
+      const message = error?.response?.data?.error
+        ? error.response.data.error
+        : error.message;
+      setNotification({ type: 'error', message });
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  let snackbar;
+  if (notification) {
+    snackbar = (
+      <Snackbar
+        anchorOrigin={{ vertical: 'bottom', horizontal: 'center' }}
+        open={!!notification}
+        autoHideDuration={5000}
+        onClose={handleCloseNotification}
+      >
+        <Alert
+          onClose={handleCloseNotification}
+          severity={notification.type}
+          sx={{ width: '100%' }}
+        >
+          {notification.message}
+        </Alert>
+      </Snackbar>
+    );
+  }
+
   return (
     <RegistrationFormWrapper>
-      <BorderForm>
-        <FormControl sx={{ width: '35ch' }}>
-          <WrapH1>
-            <Typography sx={{ fontSize: '24px' }}>Forgot password?</Typography>
-          </WrapH1>
-
-          <Box sx={{ mt: '20px' }}>
-            <TextField
-              placeholder="Please enter your email"
-              label="Email"
-              autoComplete="current-email"
-              fullWidth
-            />
-          </Box>
-
-          <WrapButtonAndText>
-            <Button variant="contained">sent password</Button>
-
-            <StyledSpan>
-              <span>I remember password</span>
-            </StyledSpan>
-          </WrapButtonAndText>
-        </FormControl>
-      </BorderForm>
+      <Box component="form" onSubmit={sendEmail}>
+        <BorderForm>
+          <FormControl sx={{ width: '35ch' }}>
+            <WrapH1>
+              <Typography sx={{ fontSize: '24px' }}>
+                Forgot password?
+              </Typography>
+            </WrapH1>
+
+            <Box sx={{ mt: '20px' }}>
+              <TextField
+                type="email"
+                id="email"
+                required
+                value={email}
+                placeholder="Please enter your email"
+                label="Email"
+                autoComplete="current-email"
+                fullWidth
+                onChange={e => setEmail(e.target.value)}
+              />
+            </Box>
+            <WrapButtonAndText>
+              <LoadingButton
+                loading={loading}
+                type="submit"
+                variant="contained"
+              >
+                sent password
+              </LoadingButton>
+              <StyledSpan>
+                <span>I remember password</span>
+              </StyledSpan>

```

Comment:
add a link to login onClick remember password

Reply:
I got this layout, I'll fix it